### PR TITLE
VIH-7212 enable service web for mobile tablet

### PR DESCRIPTION
**Pull request by omio-ma**
### JIRA link ###
https://tools.hmcts.net/jira/browse/VIH-7212

### Change description ###
- enabling service web for android/tablets

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -205,6 +207,8 @@ parameters:
       value: $(KinlySelfTestApiUrl)/testcall
     - name: VhServices:PexipSelfTestNodeUri
       value: $(PexipSelfTestNode)
+    - name: VhServices:EnableMobileSupport
```
Comment (lowflying) [inline azure-pipelines.yml:210]: I know I said this was ok yesterday but could you change this to be the same as the CD pipeline? CI runs will then ensure any issues with the Library set up are caught early.

Comment (lowflying) [reply azure-pipelines.yml:210]: Awesome, thank you!

**Review by lowflying — APPROVED** [review]
I'm happy with the Pipeline bits anyway

**Review by ZJianWang — APPROVED** [review]

**Pull request merged**